SpatialGDK/Source/Private/Interop/GlobalStateManager.cpp

```diff
@@ -188,3 +256,137 @@ bool UGlobalStateManager::IsSingletonEntity(Worker_EntityId EntityId)
 	}
 	return false;
 }
+
+void UGlobalStateManager::ToggleAcceptingPlayers(bool bInAcceptingPlayers)
+{
+	if (!bHasLiveMapAuthority || !Cast<USpatialGameInstance>(NetDriver->GetWorld()->GetGameInstance())->bIsWorkerAuthorativeOverGSM)
+	{
+		UE_LOG(LogGlobalStateManager, Error, TEXT("Tried to toggle AcceptingPlayers but this worker is not authorative over the GSM"));
+	}
+
+	// GSM Constraint
+	Worker_Constraint GSMConstraint;
+	GSMConstraint.constraint_type = WORKER_CONSTRAINT_TYPE_COMPONENT;
+	GSMConstraint.component_constraint.component_id = SpatialConstants::GLOBAL_STATE_MANAGER_MAP_URL;
+
+	// GSM Query
```

Comment:
Could `QueryGSM` be used somehow to reduce code duplication?

Reply:
Querying removed, code reduced.